src/app/components/client/stories/Chart.stories.ts

```diff
@@ -0,0 +1,41 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+import type { Meta, StoryObj } from "@storybook/react";
+
+import { DoughnutChart } from "../Chart";
+
+import styles from "../Chart.module.scss"
+
+// More on how to set up stories at: https://storybook.js.org/docs/react/writing-stories/introduction
+const meta: Meta<typeof DoughnutChart> = {
+  title: "Charts",
+  component: DoughnutChart,
+};
+export default meta;
+type Story = StoryObj<typeof DoughnutChart>;
+
+// Organized from $color-purple-90 to $color-purple-20, refer to tokens.scss
+const purpleColors = ['#321c64', '#45278d', '#592acb', '#7542e5', '#9059ff', '#ab71ff', '#c689ff', '#cb9eff'];
+
+// Predicting that the data may result in this fashion
+const data: any[] = [
+  ['Home address', 11],
+  ['Family members', 12],
+  ['Contact Info', 1],
+  ['Full name', 5],
+  ['Other', 2]
+];
+
+const labels: string[] = data.map(([label]) => label);
+const numbers: number[] = data.map(([, number]) => number);
```

Comment:
Since the above `data` structure is basically a `Map()`, you could also do something like this; although not convinced it's any better seems like the same amount of loops/variables/complexity:

```js
const data = new Map([
  ['Home address', 11],
  ['Family members', 12],
  ['Contact Info', 1],
  ['Full name', 5],
  ['Other', 2]
]);
const labels = [...data.keys()] // ['Home address', 'Family members', 'Contact Info', 'Full name', 'Other']
const numbers = [...data.values()]; // [11, 12, 1, 5, 2]
```